content/dev/autograders.md

```diff
@@ -2,4 +2,74 @@
 
 # Autograders
 
-stub
\ No newline at end of file
+PrairieLearn supports [external autograders](https://prairielearn.readthedocs.io/en/latest/externalGrading/) for assessment of students' submissions that is highly customizable. A common use case is to assess student-submitted code against tests and give a grade and feedback based on the test results.
+
+If you are working in [Python](https://prairielearn.readthedocs.io/en/latest/python-grader/), [C or C++](https://prairielearn.readthedocs.io/en/latest/c-grader/), or [Java](https://prairielearn.readthedocs.io/en/latest/java-grader/), you will likely want to use the provided autograders.
```

Comment:
We could add R here, but it does not have a good site besides the directory in the prairielearn sources.

Reply:
Definitely add it! I think it's great to show people some of the other places they can find autograders.